generators/client/templates/client/webpack/webpack.config.dev.js

```diff
@@ -43,8 +43,6 @@ module.exports = () => ({
     clientLogLevel: 'none',
     // enable HMR on the server
     hot: true,
-    // Enable SSL
-    https: true,
```

Comment:
why remove this ?

Reply:
because hot reloading doesnt work on the opensource generator without this.

And also because it expects you to accept certificate 5 times a week @ghamaide